### ISSUE TITLE
Accessibility: Screen titles

### ISSUE THREAD

**Issue by malinajirka** [wordpress-mobile/WordPress-Android#7370]
Audit screen titles - screen title should be announced by TalkBack when a screen is shown. Make sure that each screen has a title, because otherwise the TalkBack announces the name of the Application instead.
